tests/test_image_utils.py

```diff
@@ -367,3 +368,89 @@ def test_center_crop_tensor(self):
             # Check result is consistent with PIL.Image.crop
             cropped_image = feature_extractor.center_crop(image, size)
             self.assertTrue(torch.equal(cropped_tensor, torch.tensor(feature_extractor.to_numpy_array(cropped_image))))
+
+
+@require_vision
+class LoadImageTester(unittest.TestCase):
+    def test_load_img_local(self):
+        img = load_image("./tests/fixtures/tests_samples/COCO/000000039769.png")
+        img_arr = np.array(img)
+
+        self.assertEqual(
+            img_arr.shape,
+            (480, 640, 3),
+        )
+
+    def test_load_img_url(self):
+        img = load_image("http://images.cocodataset.org/val2017/000000039769.jpg")
```

Comment:
Not sure if we can let our tests depend on an external URL, cc @LysandreJik

Reply:
Could we move it in our datasets for the tests? That would be more reliable.

Reply:
oh I see. I've just removed this test method entirely since the goal was to test external URL